__tests__/Apps.js

```diff
@@ -98,8 +104,19 @@ describe('Apps and App Catalog', () => {
 
         expect(introText).toBeInTheDocument();
 
+        const main = await findByTestId('main');
```

Comment:
I did this so the debug output would be a bit more constrained when this test fails.